stl/inc/any

```diff
@@ -9,7 +9,9 @@
 #include <yvals_core.h>
 #if _STL_COMPILER_PREPROCESSOR
 
-#if _HAS_CXX17
+#if !_HAS_CXX17

```

Comment:
I am usually the person who hates inverted conditionals with else, but I think this is the exception that proves the rule :) Thanks!